adobe/aepsdk-edgeidentity-ios
# `dev` -> `staging` [NON-RELEASE] Migrate to GitHub Actions reusable workflows

## Description

This merge contains updates to migrate to GitHub Actions reusable workflows in aepsdk-commons using tag `gha-ios-5.0.0`.

### Dependency chanages
* Updates the gem bundler from `1.17.2` -> `2.5.23` (Gemfile.lock)
    * Addresses a build failure from incompatibility with bundler `1.17.2` and ruby `3.x`: https://github.com/adobe/aepsdk-edgeidentity-ios/actions/runs/12020105024/job/33508294774
* Update cocoapods to `1.16.2` (latest) - Podfile.lock, Gemfile, Gemfile.lock

### Testing changes
* Update AEPTestUtils to 5.2.2 (Core)
    * Update deprecated usage of EventHub.reset to MobileCore.resetSDK

See PR:
* #134

## Related Issue






## Motivation and Context



## How Has This Been Tested?





## Screenshots (if appropriate):

## Types of changes




## Checklist:
